base/services/nfsd/daemon_debug.c

```diff
@@ -228,45 +228,45 @@ void print_share_mode(int level, DWORD mode)
 void print_file_id_both_dir_info(int level, FILE_ID_BOTH_DIR_INFO *pboth_dir_info)
 {
     if (level > g_debug_level) return;
-    fprintf(dlog_file, "FILE_ID_BOTH_DIR_INFO %p %d\n", 
+    fprintf(dlog_file, "FILE_ID_BOTH_DIR_INFO %p %Id\n", 
```

Comment:
I think this is in sync with upstream. Perhaps disable the warning via CMake?

Reply:
Yeah, I guess since this is debug code, we can get away with it (though using %x for 64 bit values is really broken). I thought about an upstream patch, but since ```%Ix``` is Windows specific and ```%zx``` is not supported on ReactOS yet, it would still require a patch. And the repo on GitHub looks unmaintained. So I'll do it the lazy way.